### PR TITLE
Made the section Playground section responsive

### LINKED ISSUE #6069: Responsive issue on homepage
### Description

on homepage https://layer5.io
Screenshot_20241108-173614

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by NishantSinghhhhh**
Description

This PR fixes #6069 

Notes for Reviewers

This PR includes updates to make one section of the home page responsive. The changes include:

Added media queries to adjust layout and styling for smaller screen sizes.
Adjusted padding, margin, and font sizes to enhance readability and spacing on mobile devices.
Updated the CSS for images and containers in the section to ensure proper scaling and alignment on screens below 768px and 470px.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5f76aec7933cae1e58c22b3aeea359a4240ebe7f at: https://6730e2ba4020f1fd389b1b9f--layer5.netlify.app

File: src/sections/Home/Playground-home/index.js [hunk 1]
```diff
@@ -65,18 +65,32 @@ const ViewsSectionWrapper = styled.div`
       text-align: center;
       flex-direction: column-reverse;
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:66]:
```suggestion
      flex-direction: column-reverse;
      height: 600px;
      padding: 0 2%;
```

File: src/sections/Home/Playground-home/index.js [hunk 2]
```diff
@@ -65,18 +65,32 @@ const ViewsSectionWrapper = styled.div`
...
         display: flex;
         flex-direction: column;
         flex: 0 0 50%;
         max-width: 50%;
         padding-bottom: 3rem;
+        padding-left: 1rem;
         @media only screen and (max-width: 767px) {
           max-width: 100%;
           justify-content: center;
           text-align: center;
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:87]:
```suggestion
          text-align: center;
          margin-top: 4rem;
```

File: src/sections/Home/Playground-home/index.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ const ViewsSectionWrapper = styled.div`
     height: 500px;
     justify-content: space-evenly;
     align-items: center;
-    padding: 0 5% 0 0;
+    padding: 0 0 0 0;
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:58]:
```suggestion
    padding: 0 5% 0 0;
```

File: src/sections/Home/Playground-home/index.js [hunk 4]
```diff
@@ -65,18 +65,32 @@ const ViewsSectionWrapper = styled.div`
...
       flex-direction: column-reverse;
     }
+      
+    @media only screen and (max-width: 450px) {
+      height: 600px;
+    }
+
+    @media only screen and (max-width: 296px) {
+      height: 800px;
+    }
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:75]:
```suggestion

```

File: src/sections/Home/Playground-home/index.js [hunk 5]
```diff
@@ -65,18 +65,32 @@ const ViewsSectionWrapper = styled.div`
...
         @media only screen and (max-width: 767px) {
           max-width: 100%;
           justify-content: center;
           text-align: center;
         }
+
+        @media only screen and (max-width: 450px) {
+          margin-top: 200px;
+        }
+
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:93]:
```suggestion
```

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
@NishantSinghhhhh please consider incorporating suggested changes.

File: src/sections/Home/Playground-home/index.js [hunk 6]
```diff
@@ -65,18 +65,32 @@ const ViewsSectionWrapper = styled.div`
...
+      height: 800px;
+    }
 }
     .hero-text {
         display: flex;
         flex-direction: column;
         flex: 0 0 50%;
         max-width: 50%;
         padding-bottom: 3rem;
+        padding-left: 1rem;
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:83]:
```suggestion

```

File: src/sections/Home/Playground-home/index.js [hunk 7]
```diff
@@ -76,7 +79,13 @@ const ViewsSectionWrapper = styled.div`
           max-width: 100%;
           justify-content: center;
           text-align: center;
+          margin-top: 4rem;
+        }
+
+        @media only screen and (max-width: 450px) {
+          margin-top: 200px;
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:86]:
```suggestion
```

File: src/sections/Home/Playground-home/index.js [hunk 8]
```diff
@@ -76,7 +79,11 @@ const ViewsSectionWrapper = styled.div`
           max-width: 100%;
           justify-content: center;
           text-align: center;
+          margin-top: 4rem;
+        }
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:83]:
```suggestion
 
```

File: src/sections/Home/Playground-home/index.js [hunk 9]
```diff
@@ -76,7 +79,11 @@ const ViewsSectionWrapper = styled.div`
           max-width: 100%;
           justify-content: center;
           text-align: center;
+          margin-top: 4rem;
+ 
+
```
Comment (vishalvivekm) [inline src/sections/Home/Playground-home/index.js:84]:
```suggestion
```

File: src/sections/Home/Playground-home/index.js (hunk 4 above)
Comment (NishantSinghhhhh) [reply src/sections/Home/Playground-home/index.js]: Sir after 300px too, this error is coming in which the text is not visible , so for that , we need to add the media query for 296px

**Comment by l5io** [comment]
🚀 Preview for commit ae9bd7a57ef4ef9624783289190195a58d25f8eb at: https://6731002ae3f3370614e4a348--layer5.netlify.app

File: src/sections/Home/Playground-home/index.js (hunk 4 above)
Comment (vishalvivekm) [reply src/sections/Home/Playground-home/index.js]: @NishantSinghhhhh take a look at the latest deploy preview: https://6731002ae3f3370614e4a348--layer5.netlify.app/

File: src/sections/Home/Playground-home/index.js (hunk 6 above)
Comment (vishalvivekm) [reply src/sections/Home/Playground-home/index.js]: added @NishantSinghhhhh 
https://github.com/layer5io/layer5/pull/6071/commits/28306ed689d736e860fcb8f0b40059f599d37249

**Comment by l5io** [comment]
🚀 Preview for commit 28306ed689d736e860fcb8f0b40059f599d37249 at: https://6731086b88db9e2f474094e1--layer5.netlify.app

**Review by vishalvivekm — APPROVED** [review]

**Comment by vishalvivekm** [comment]
@NishantSinghhhhh Thank you for your contribution!
Let's discuss this during the website call on Monday at 6:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Pull request merged**